### DISCUSSION TITLE
We found that -useperfthreads improves performance on Windows, but has a negative impact on FPS on Linux

### DISCUSSION THREAD

**Discussion by NepPure** [thijsvanloef/palworld-server-docker#232] [General]
As title.

I want to know how everyone tested this configuration.

https://github.com/thijsvanloef/palworld-server-docker/blob/14b28694bd897bcac8ce23943ef6a2a0481605e2/scripts/start.sh#L23

**Comment by thijsvanloef** [comment]
Hi @NepPure, is this something you've tested on the same hardware?
